modules/subprocess/src/main/scala/notebook/kernel/pfork/BetterFork.scala

```diff
@@ -170,7 +169,7 @@ object BetterFork {
         } else {
           acc ++ (cl.asInstanceOf[URLClassLoader].getURLs map { u =>
             val f = new File(u.getFile)
-            URLDecoder.decode(f.getAbsolutePath, "UTF8")
+            URLDecoder.decode(f.getParent, "UTF8")
```

Comment:
Hey, no worries. My hack works, it's just not as elegant. For now I am happy with this, but I am interested in solving this issue elegantly.

Nice, so you are in the bay? Very cool! I hope you had a great stay. I would be thrilled too! 

Apart from that, I like spark-notebook very much. The standard plotting is a bit weird (okay granted, I haven't figured it out!!) but I used WISP and its great! I think we should work on documentation, as it is outdated. Also, I couldn't get D3 to work, because of some 'Codec' chaos... but that's all for the future, and shows me, that there is work to be done - in general, bigdata still has a lot to accomplish in terms of usability. Spark-Notebook is a light on the horizon. Great work, andy!

greets, carsten